utils/brand.env

```diff
@@ -1,7 +1,9 @@
+export SEARX_URL=''
```

Comment:
Since this file is built each time it is required using `make`, is it required to commit this file ? or can we remove this file and add it to .gitignore ?

Reply:
My aim is to remove utils/brand.env entirely .. but this will take some days, since I am back at my customers these days.

Reply:
Implemented in the branch https://github.com/dalf/searx/commits/fix-rm-brand-make

feel free to cherry-pick

Reply:
Thanks for a669c8a4 but I think this will not be enough.  The reason brand.env exists is because it is sourced by the shell scripts:

    ./utils/filtron.sh:9:source "${REPO_ROOT}/utils/brand.env"
    ./utils/searx.sh:9:source "${REPO_ROOT}/utils/brand.env"
    ./utils/morty.sh:8:source "${REPO_ROOT}/utils/brand.env"